src/main/java/hudson/plugins/sonar/client/HttpClient.java

```diff
@@ -37,12 +39,12 @@ public String getHttp(String urlToRead, String username, String password) throws
       if (!StringUtils.isEmpty(password)) {
         userpass = userpass + password;
       }
-      String basicAuth = "Basic " + javax.xml.bind.DatatypeConverter.printBase64Binary(userpass.getBytes(StandardCharsets.UTF_8));
```

Comment:
With this plugin update, there are some build tools that have been integrated like [animal-sniffer](http://www.mojohaus.org/animal-sniffer/). The baseline being used is 1.580.3 where Java7 is not supported. Because the baseline is 1.580.3, the java level is set to Java6 and animal-sniffer performs the check at that level. [StandardCharsets](https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html) do not exist in Java6 so animal-sniffer throw an error and breaks the build.

Possible solutions I can come up with are:

- Bump baseline to 1.642+ and remove `<java.level>6</java.level>`. (Default is Java7)
- Use Google's Charset to avoid this problem.

Chose second option to be less intrusive.